gems/pending/util/vcenter_thumb_print.rb

```diff
@@ -0,0 +1,25 @@
+require 'util/thumb_print'
+
+class VcenterThumbPrint < ThumbPrint
+  def initialize(host)
+    super(host)
+  end
```

Comment:
why do we need this `initialize` in the subclass if it only calls the one in the base class?